app/src/main/java/com/devindi/wallpaper/settings/SettingsAdapter.kt

```diff
@@ -0,0 +1,25 @@
+package com.devindi.wallpaper.settings
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.devindi.wallpaper.R
+import com.devindi.wallpaper.settings.model.IntField
+import com.devindi.wallpaper.settings.size.SizeViewHolder
+import com.devindi.wallpaper.source.OnItemClickListener
+
+class SettingsAdapter(private val items: List<IntField>, private val clickListener: OnItemClickListener) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/f22b9ee68b14344acb1bd6cd546c2767866b4eef/app/src/main/java/com/devindi/wallpaper/settings/SettingsAdapter.kt#L11"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>